jupyterhub/handlers/pages.py

```diff
@@ -72,6 +72,7 @@ class SpawnHandler(BaseHandler):
             spawner_options_form=spawner_options_form,
             error_message=message,
             url=self.request.uri,
+            spawner=user.spawner
```

Comment:
careful, this should be `for_user.spawner`

Reply:
Feel free to add a comment clarifying that 'user' is the authenticated user making the request and 'for_user' is the user whose server is being spawned.

Reply:
Hey @minrk if that's the case, do we actually need to bother at all?  We're already passing `for_user` through to be rendered.  I just checked, {{ for_user.spawner.whatever }} works...

Reply:
let's pass spawner anyway because I think eventually we are going to need to use this form for named servers, where user.spawner isn't specific enough.